release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2733,6 +2733,51 @@ This section will continue to be updated over time to provide notes on enhanceme
 ====
 
+[id="ocp-4-15-23_{context}"]
+=== RHSA-2024:4699 - {product-title} 4.15.23 bug fix and security update
+
+Issued: 2024-07-25
+
+{product-title} release 4.15.23 is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2024:4699[RHSA-2024:4699] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHSA-2024:4702[RHSA-2024:4702] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.23 --pullspecs
+----
+
+[id="ocp-4-15-23-enhancements_{context}"]
+==== Enhancements
+
+The following enhancement is included in this z-stream release:
+
+[id="ocp-4-15-23-timeout-ingress_{context}"]
+===== Adding connectTimeout tuning option to Ingress Controller API
+
+* The IngressController API is updated with a new tuning option, `ingresscontroller.spec.tuningOptions.connectTimeout`, which defines how long the router will wait for a response when establishing a connection to a backend server. (link:https://issues.redhat.com/browse/OCPBUGS-36208[*OCPBUGS-36208*])
+
+[id="ocp-4-15-23-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the OpenSSL versions for the Machine Config Operator and the hosted  control plane were not the same. With this release, the FIPS cluster `NodePool` creation for {product-title} 4.14 and {product-title} 4.15 has been fixed and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-37266[*OCPBUGS-37266*])
+
+* Previously, the operand details displayed information for the first custom resource definition (CRD) that matched by name. With this release, the operand details page displays information for the CRD that matches by name and the version of the operand. (link:https://issues.redhat.com/browse/OCPBUGS-36971[*OCPBUGS-36971*])
+
+* Previously, the HyperShift hosted control plane (HCP) would fail to generate ignition because of mismatched versions of {op-system-base-full} OpenSSL versions used by the HyperShift Control Plane Operator and the Machine Config Operator. With this release, the versions of {op-system-base-full} OpenSSL match correctly and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36863[*OCPBUGS-36863*])
+
+* Previously, the Ingress Operator could not successfully update the canary route because the Operator did not have permission to update `spec.host` or `spec.subdomain` on an existing route. With this release, the required permission is added to the cluster role for the Operator's ServiceAccount and the Ingress Operator can update the canary route. (link:https://issues.redhat.com/browse/OCPBUGS-36466[*OCPBUGS-36466*])
```

Comment:
```suggestion
* Previously, the Ingress Operator could not successfully update the canary route because the Operator did not have permission to update `spec.host` or `spec.subdomain` on an existing route. With this release, the required permission is added to the cluster role for the Operator's `ServiceAccount` and the Ingress Operator can update the canary route. (link:https://issues.redhat.com/browse/OCPBUGS-36466[*OCPBUGS-36466*])
```